src/sentry/static/sentry/app/views/settings/organization/navigationConfiguration.jsx

```diff
@@ -74,6 +73,12 @@ const organizationNavigation = [
         show: ({access}) => access.has('org:integrations'),
         description: t('Manage integrations for an organization'),
       },
+      {
+        path: `${pathPrefix}/developer-settings/`,
```

Comment:
Is `developer-settings` added to the project slugs block list?

Reply:
This is under settings isn't it?

Reply:
Project settings is: `/settings/<org>/<project>/`

Reply:
the develop settings are only on the org though - so should it be added to project settings?

Reply:
Because of the project settings route, org settings routes need to be added to `RESERVED_PROJECT_SLUGS` in (`src/sentry/constants.py`) - otherwise if someone named their project `developer-settings`, they wouldn't be able to access their project settings.

Reply:
I checked and there are no projects called `developer-settings` currently.

Reply:
cool will add to block list